### ISSUE TITLE
CodeArea lacks colored focus indicator

### ISSUE THREAD

**Issue by hwaite** [TomasMikula/RichTextFX#202]
Unlike standard JavaFX controls, CodeArea fails to display highlighted border when focused.

    public class Test extends Application {
        public static void main(String[] args) {launch(args);}

        public void start(Stage pStage) {
            pStage.setTitle("Test");

            final Pane root = new VBox();

            root.getChildren().addAll(
                new CodeArea(), new TextArea(), new TextField()
            );

            pStage.setScene(new Scene(root, 300, 250));
            pStage.show();
        }
    }

**Comment by TomasMikula** [comment]
Not sure we should have that highlighting by default. You can add highlighting by

```css
.code-area:focused {
	-fx-border-color: -fx-focus-color;
	-fx-border-width: 2;
	-fx-border-radius: 3;
}
```

You can study `modena.css` and try to make it match JavaFX's TextArea's border exactly.

**Comment by hwaite** [comment]
OK, thx.
